Dfe.PrepareConversions/Dfe.PrepareConversions.Tests/Mappings/CreateSponsoredProjectMapperTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AutoFixture;
+using Dfe.PrepareConversions.Data.Models.Establishment;
+using Dfe.PrepareConversions.Data.Models.Trust;
+using Dfe.PrepareConversions.Mappings;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using System;
+using Xunit;
+
+namespace Dfe.PrepareConversions.Tests.Mappings;
+
+public class CreateSponsoredProjectMapperTests
+{
+   [Fact]
+   public void Should_Map_Establishment_With_PfiScheme_And_Trust_Into_Dto_CreateSponsoredProject()
+   {
+      // Arrange
+      var fixture = new AutoFixture.Fixture();
```

Comment:
If we're newing up the fixture in each test is it worth doing it at class level

Reply:
well, there's a section of people that includes me, who believe that the setup for a test belongs within the test. I prefer it this way if the fixture instance isn't going to be shared across tests